presentation/testing/src/presentation-testing/IModelUtilities.ts

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module IModel
+ */
+
+import { IModelDb, IModelJsFs, SnapshotDb } from "@itwin/core-backend";
+import { Id64String } from "@itwin/core-bentley";
+import { BisCodeSpec, Code, CodeScopeProps, CodeSpec, ElementAspectProps, ElementProps, LocalFileName, ModelProps } from "@itwin/core-common";
+import { IModelConnection, SnapshotConnection } from "@itwin/core-frontend";
+import path from "path";
+import { getTestOutputDir } from "./Helpers";
+
+/**
+ * Interface for IModel builder pattern. Used for building IModels to test rulesets.
+ *
+ * @internal
+ * @beta
```

Comment:
Can't be both @beta and @internal. The former says we're planning to make API public in the future, the latter - we're not.